src/codemodder/scripts/generate_docs.py

```diff
@@ -224,6 +224,14 @@ class DocMetadata:
     ),
 }
 
+METADATA = CORE_METADATA | {
+    "numpy-nan-equality-S6725": DocMetadata(
```

Comment:
Is this the actual Sonar rule ID?

Reply:
`S6725` or more precisely `python:S6725` is the id for the rule. I'm following the naming convention adopted by `codemodder-java`.